frontend/utilities/priceUtils.ts

```diff
@@ -0,0 +1,12 @@
+export function dollarsToCents(dollars: number): number {
+  return Math.round(dollars * 100);
+}
+
+export function centsToDollars(cents: number): number {
+  return parseFloat((cents / 100).toFixed(2));
+}
+
+export function displayPrice(price: number): number {
```

Comment:
Put a short doc above this function seeing that this is used extensively throughout the codebase.